source/includes/_opportunities.md

```diff
@@ -232,7 +232,7 @@ This endpoint returns all opportunities you have Created.
 
 `GET https://api.apollo.io/v1/opportunities`
 
-## View Opportunity
+## View opportunity
```

Comment:
Line no 231 
`This endpoint returns all opportunities you have Created.` -> `This endpoint returns all opportunities you have created.`